ompi/mca/osc/rdma/osc_rdma_component.c

```diff
@@ -223,6 +226,13 @@ static int ompi_osc_rdma_component_register (void)
                                             MCA_BASE_VAR_TYPE_STRING, NULL, 0, 0, OPAL_INFO_LVL_3,
                                             MCA_BASE_VAR_SCOPE_GROUP, &ompi_osc_rdma_btl_names);
 
+    ompi_osc_rdma_mtl_names = "psm2";
+    (void) mca_base_component_var_register (&mca_osc_rdma_component.super.osc_version, "mtls",
+                                            "Comma-delimited list of MTL component names to lower the priority of rdma "
+                                            "osc component favoring pt2pt osc (default: psm2)",
```

Comment:
You should probably `asprintf()` the help string to show the the default value as whatever is in `ompi_osc_rdma_mtl_names` (because someone will inevitably modify that value someday and forget to update this help string).